### PR TITLE
Edit button, delete userinfo

### PR DESCRIPTION
**Pull request by sungbae**

### CODE REVIEW

File: static/javascripts/components/survey_card.jsx [hunk 1]
```diff
@@ -121,7 +121,6 @@ var SurveysPage = React.createClass({
                   <QuestionDiv questions={this.state.questions} updateQuestions={this.updateQuestions} />
                   <AddQuestion onAdding={this.handleAdding}/>
                   <FinishSurvey onSubmit={this.handleSubmit}/>
-                  </div>
```
Comment (sungbae) [inline static/javascripts/components/survey_card.jsx:124]: There was an extra div

File: static/javascripts/components/profile.jsx [hunk 2]
```diff
@@ -49,65 +37,88 @@ var ProfilePage = React.createClass({
...
+            Click to toggle. {edit_state}.
+          </p>
+          <li>Username: {this.state.username}</li>
+          <li>Affiliation: {this.state.user_affiliation}</li>
+          <li>Email: {this.state.email}</li>
+          <li>Birth Date: {this.state.dob}</li>
+          <li>Gender: {this.state.user_gender}</li>
+          <li>Ethnicity: {this.state.user_ethnicity}</li>
+          <li>Native Language: {this.state.user_native_language}</li>
+          <li>Academic Year: {this.state.user_status}</li>
```
Comment (sungbae) [inline static/javascripts/components/profile.jsx]: Would really appreciate ideas on how to separate this field out if the user is a faculty. Obviously, I can't put an if statement in the return.

Comment (Brady76) [inline static/javascripts/components/profile.jsx]: if(faculty == true){
  return(  
    username through native language
  );

else
  return(  
    username through academic year  
  );

Comment (sungbae) [inline static/javascripts/components/profile.jsx]: @Bradyta So just put another conditional with duplicate code?

Comment (Brady76) [inline static/javascripts/components/profile.jsx]: Basically, yeah.

**Comment by sungbae** [comment]
I have finished this PR. @Bradyta @nicot @untra , does anyone want to look over and merge it? I have tested this out plenty of times but better safe than sorry I guess....

**Comment by nicot** [comment]
Looks good to me!

**Pull request merged**